db/db.sql

```diff
@@ -0,0 +1,36 @@
+CREATE DATABASE messstation;
```

Comment:
CREATE DATABASE IF NOT EXISTS `messstation` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci;
USE messstation